scene/resources/packed_scene.h

```diff
@@ -137,7 +137,7 @@ class SceneState : public RefCounted {
 	void clear();
 
 	bool can_instance() const;
```

Comment:
For instance:

```suggestion
	bool can_instantiate() const;
```

But can also rename rename this to `is_instantiable()`, depending on the context.